src/components/EmbeddedAskForm.scss

```diff
@@ -42,12 +53,3 @@
     display: none;
   }
 }
-
-.form-toggle-button {
-  background: #4a90e2;
-  border-radius: 1em;
-  color: white;
-  display: inline-block;
-  padding: 5px 10px;
-  margin: 2em 0;
-}
```

Comment:
@isaacdurazo Why did you delete this, by the way? I'm reinstating it as I fix up #55